redis/gradle.properties

```diff
@@ -0,0 +1,2 @@
+version = 0.2-SNAPSHOT
```

Comment:
Since it will be the first release of this module, this should be `0.1-SNAPSHOT`. The modules in this repo released independently.

Reply:
Ok.